### PR TITLE
Added Google Sign-in

### PR DESCRIPTION
**Pull request by xstoop**

### CODE REVIEW

**Comment by m1guelpf** [comment]
Please add to docs

**Comment by xstoop** [comment]
Ok,I didn't found need add docs  in the contribute page https://socialiteproviders.netlify.com/contribute.html.

Otherwise i have a question,if I need to remove the Google+ providers? Google+ will shut down soon .In Google+  provider class use the `const IDENTIFIER='google'` and `namespace SocialiteProviders\Google`, but it using the Google-Plus dirname. In this PT,i just created a new file to add Google sign-in and using the `const IDENTIFIER='google'` and `namespace SocialiteProviders\Google` same of the google+ provider.

About the doc, the google+ provider's doc has used the `google.md` filename, but the  title in the doc is `google+`.So i need to edit this `google.md` content or i just change the `google.md` name to `google+.md` and create a new `google.md` file?

How can i do ?

File: src/Google/Provider.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        return 'https://oauth2.googleapis.com/token';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getUserByToken($token)
+    {
+        $response = $this->getHttpClient()->get(
+            'https://www.googleapis.com/plus/v1/people/me', [
```
Comment (CSNWEB) [inline src/Google/Provider.php]: This seems to be the old google plus api that is going to be deprecated soon?

File: src/Google/Provider.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'GOOGLE';
```
Comment (CSNWEB) [inline src/Google/Provider.php]:
```suggestion
    const IDENTIFIER = 'GOOGLEOAUTH';
```

File: src/Google/Provider.php (hunk 1 above)
Comment (CSNWEB) [inline src/Google/Provider.php]:
```suggestion
            'https://www.googleapis.com/oauth2/v3/userinfo', [
```
Got it working with this and the mapping changes below

**Comment by xstoop** [comment]
@m1guelpf https://github.com/SocialiteProviders/Documentation/pull/34

File: src/Google/Provider.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider implements ProviderInterface
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    const IDENTIFIER = 'GOOGLEOAUTH';
```
Comment (antonkomarev) [inline src/Google/Provider.php]: Why it doesn't follow same convention with another providers? It should be:
```
const IDENTIFIER = 'GOOGLE';
```

Comment (antonkomarev) [reply src/Google/Provider.php]: Ah... I see.. that's because GooglePlus has wrong identifier :\

**Pull request merged**